src/sdl2_compat.c

```diff
@@ -1833,6 +1850,23 @@ Event2to3(const SDL2_Event *event2, SDL_Event *event3)
     case SDL_EVENT_JOYSTICK_BATTERY_UPDATED:
         /* This should never happen, but see Event3to2() for details */
         break;
+    case SDL_EVENT_AUDIO_DEVICE_ADDED:
+        if (event2->adevice.iscapture) {
+            if (event2->adevice.which >= 0 &&
```

Comment:
this `which>=0`check is bogus: which member is an Uint32. (same with the one below.) Should we remove?

Reply:
Yes please.

Reply:
Done